xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/objects/classes/TextAreaClass.java

```diff
@@ -346,6 +347,37 @@ public boolean isWikiContent()
         }
     }
 
+    /**
+     * Indicate if the content of this property should be executed in a restricted content (provided the type indicate
+     * that this content should be executed).
+     * 
+     * @return true if the content of this property should be executed in a restricted content, false otherwise
+     * @since 14.10RC1
+     * @since 14.4.7
+     * @since 13.10.11
+     */
+    @Unstable
+    public boolean isRestricted()
+    {
+        return (getIntValue("restricted", 0) == 1);
```

Comment:
Minor: superfluous wrapping parenthesis.

Reply:
Indeed, I copy/pasted it from `PropertyClass#isDisabled` (yes, I'm lazy like that) without noticing.